numpy/lib/histograms.py

```diff
@@ -308,7 +351,7 @@ def _get_bin_edges(a, bins, range, weights):
             n_equal_bins = 1
         else:
             # Do not call selectors on empty arrays
-            width = _hist_bin_selectors[bin_name](a)
+            width = _hist_bin_selectors[bin_name](a, range)
```

Comment:
This should pass `range=(first_edge, last_edge)`, which is the processed range argument

Reply:
do you mean `range(first_edge, last_edge)`?